components/navigation/ui/src/main/resources/XWiki/SearchSuggestConfig.xml

```diff
@@ -970,7 +970,7 @@
       <resultsNumber>5</resultsNumber>
     </property>
     <property>
-      <url>$xwiki.getURL('PhenoTips.FamilySearch', 'get', 'outputSyntax=plain&amp;permission=view')</url>
+      <url>${request.contextPath}/rest/families/suggest?permission=view</url>
```

Comment:
The global search doesn't work anymore. Trying to write something in that input shows a modal exception (`Not acceptable`) because the search widget requests `application/xml`, and the rest services can only produce JSON. Then, even if I get it to request JSON instead, nothing is actually displayed since the search widget expects a specific kind of response, which isn't compatible with the output generated by the patients/families suggest REST services.